pcsx2/gui/AppMain.cpp

```diff
@@ -529,8 +530,9 @@ void DoFmvSwitch(bool on)
 				viewport->DoResize();
 	}
 
-	if (EmuConfig.Gamefixes.FMVinSoftwareHack) {
+	if (EmuConfig.Gamefixes.FMVFix) {
 		ScopedCoreThreadPause paused_core(new SysExecEvent_SaveSinglePlugin(PluginId_GS));
+		g_Conf->EmuOptions.Speedhacks.EECycleRate = 0;
```

Comment:
If I close PCSX2 in the middle of an FMV with the hack enabled, the next time I reopen it the EE cyclerate will be set to 0 instead of whatever it was originally at. It might be better to avoid changing the ini value so unexpected changes do not occur.

Reply:
What value do I need to access to change the EE cyclerate without touching the settings themselves?

Reply:
Another potentially undesirable behavior of the rendering switch is that if the plugin is set to use software rendering by default, it will switch to hardware rendering.

Is there a value I can check  to avoid the switch when it is already in software mode?

Reply:
> What value do I need to access to change the EE cyclerate without touching the settings themselves?

Probably EmuConfig.Speedhacks.EECycleRate. Maybe adjusting `fixup` in `_ApplySettings` or related code would work (I haven't put much thought into this btw).

> Is there a value I can check to avoid the switch when it is already in software mode?

No. Doesn't currently exist.